qcodes/instrument_drivers/Keysight/private/Keysight_344xxA_submodules.py

```diff
@@ -438,7 +444,12 @@ def __init__(self, name: str, address: str, silent: bool = False,
         ####################################
         # Instrument specifications
 
-        self.has_DIG = 'DIG' in self._licenses()
+        options = self._options()
+        self.has_DIG = self.is_34465A_34470A and (
+            'DIG' in options
+            or LooseVersion('A.03') <= LooseVersion(idn['firmware'])
```

Comment:
Do we still need this version check?

Reply:
yes, because  as per our testing `options` does NOT return DIG for 3.0 firmware :)

Reply:
That is disappointing. Please document in a comment

Reply:
yes, done.